### DISCUSSION TITLE
Developing pomotroid2 informally.

### DISCUSSION THREAD

**Discussion by ahmetcanisik** [Splode/pomotroid#284] [Ideas]
Hello, as you know, `pomotroid` has not received an update for a long time. I thought about it and decided that a new application would be useful as a continuation of this application, so I started developing `pomotroid2`. If you think that this project legally violates the rules, please contact me privately.

**Comment by Berkoksen** [comment]
Hi Ahmet, can you share the project if it's done?

**Reply by Splode** [reply]
I'd like to see it, too 😉
